client/rhel/spacewalk-client-tools/src/firstboot-legacy-rhel5/rhn_review_gui.py

```diff
@@ -32,7 +32,7 @@ class RhnReviewWindow(RhnRegisterFirstbootGuiWindow, rhnregGui.ReviewSubscriptio
     runPriority=108.9
     moduleName = _("Review Subscription")
     windowTitle = moduleName
-    shortMessage = _("Connect to Red Hat Satellite")
+    shortMessage = _("Connect to Uyuni")
```

Comment:
```suggestion
    shortMessage = _("Connect to the Server")
```

And anyway, same comment as above: I think this code is not used :-?

Reply:
Again: no idea.